src/state/slices/opportunitiesSlice/types.ts

```diff
@@ -137,3 +137,9 @@ export type GroupedEligibleOpportunityReturnType = {
   opportunityIds: OpportunityId[]
   netApy: number
 }
+
```

Comment:
Tags is meant to be a freeform way to give opportunities tags, so it's not specific to any provider. This prob wouldn't work here since that would limit them to just the idle tags.